quic/s2n-quic-core/src/recovery/bbr/recovery.rs

```diff
@@ -0,0 +1,220 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{recovery::bbr::recovery::State::*, time::Timestamp};
```

Comment:
it's a bit weird to drill all the way down to the module's path to use a enum. Did you try `use self::State::*;`?

Also I try to avoid polluting the namespace as much as possible and just match variants with a fully qualified enum. I'm not going to push on it a ton but IMO it helps contain things a bit.